src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/integrationtest/TransformationBulkScanTest.java

```diff
@@ -32,6 +29,7 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.bsp.common.utils.ResourceLoader.loadResourceAsString;
```

Comment:
Not sure I'd extract this. Also seems not very reusable.

Reply:
You may be actually right, that this is not strictly related to BSP itself.

But I have seen many places where it was used in divorce COS and at least one place here.